zulip_bots/zulip_bots/run.py

```diff
@@ -17,20 +17,27 @@
 
 
 def import_module_from_source(path, name=None):
-    if name is None:
+    if not name:
         name = splitext(basename(path))[0]
 
     if six.PY2:
         import imp
         module = imp.load_source(name, path)
-        return module
     else:
         import importlib.util
         spec = importlib.util.spec_from_file_location(name, path)
         module = importlib.util.module_from_spec(spec)
         spec.loader.exec_module(module)
-        return module
 
+    return module
+
+def is_not_equal(name, given_path_to_bot):
+    if name and given_path_to_bot:
+        path_to_bot_by_name = 'zulip_bots/bots/{bot}/{bot}.py'.format(bot=name)
+
+        if path_to_bot_by_name == given_path_to_bot:
+            return False
+    return True
```

Comment:
How about comparing name to ` os.path.splitext(os.path.basename(path_to_bot))[0]`?